app/src/main/java/ch/epfl/sdp/blindly/main_screen/MatchPageFragment.kt

```diff
@@ -224,7 +234,13 @@ class MatchPageFragment : Fragment(), CardStackListener {
         val profiles = ArrayList<Profile>()
         for (user in users) {
             profiles.add(
-                Profile(user.username!!, User.getUserAge(user)!!)
+                Profile(
+                    user.username!!,
+                    User.getUserAge(user)!!,
+                    user.description!!,
+                    user.passions!!.joinToString(", \n"),
+                    user.recordingPath!!
+                )
```

Comment:
I was wondering something when I worked on the Matching Algorithm. Wouldn't it be better to have a function called `createProfileFromUser(user: User)` in `Profile.kt` ? It would be possible to test it and it seems more intuitive to me. However, I know it works like that.

Reply:
I'll take a look at it, it's a good idea :D

Reply:
Okay so unfortunately it isn't possible, the Parcelable interface requires the parameters to be in the primary constructor

Reply:
That's sad :(